app/controllers/concerns/op_turbo/component_stream.rb

```diff
@@ -83,6 +83,10 @@ def render_error_flash_message_via_turbo_stream(**)
       render_flash_message_via_turbo_stream(**, scheme: :danger, icon: :stop)
     end
 
+    def render_aria_update_message(message:, type:, role: "alert")
+      turbo_streams << OpTurbo::StreamComponent.new(action: :aria, message:, type:, role:, target: nil).render_in(view_context)
```

Comment:
nitpick - formatting

```suggestion
      turbo_streams << OpTurbo::StreamComponent
        .new(action: :aria, message:, type:, role:, target: nil)
        .render_in(view_context)
```